src/preload/client/importer/FilesImportResolver.ts

```diff
@@ -38,7 +38,7 @@ export class FilesImportResolver {
   ): Promise<string | undefined> => {
     // check db for chronicles id matching name, if any
     const result = await this.knex("import_files")
-      .where({ filename: name })
+      .where({ filename: name, importerId: this.importerId })
```

Comment:
If no importerId, may grab files from different imports, specifically from different import directories. Since the current wikilinks do not handle duplicate file names, this could (did) result in e.g. a different image or video than in the original note being displayed (if they had the same name).